languages/java/exercises/concept/case-statement/src/test/java/PlayAnalyzerTest.java

```diff
@@ -0,0 +1,58 @@
+import org.junit.Assert;
+import org.junit.Test;
+
+public class PlayAnalyzerTest {
+
+    @Test(expected = IllegalArgumentException.class)
```

Comment:
FYI: the order of the tests is important, as we expect CLI-based students to solve them from top to bottom. As such, they should be ordered to match the tasks in the instructions, which means that these exception-based tests (which belong to the second task) should be the last tests.